duckduckgo/Android
# Add https summary

Asana Issue URL: https://app.asana.com/0/488551667048375/471193197990035

## Description
Add https status line to the privacy dashboard

NOTE: There is currently a url display bug on ddg, keep this in mind if using it for testing https://app.asana.com/0/414730916066338/494978153394812.

## Steps to Test this PR:
TEST ONE:
1. Visit a https site for example https://bbc.co.uk
1. Open the privacy dashboard
1. Ensure that the connection is listed as an "Encrypted connection" and that the shield icon is green

TEST TWO:
1. Visit a http site for example http://huffingtonpost.com
1. Open the privacy dashboard
1. Ensure that the connection is listed as an "Unencrypted connection" and that the shield icon is red

TEST THREE:
1. Visit a mixed content site for example https://ebuddy.com/
1. Open the privacy dashboard
1. Ensure that the connection is lisedt as an "Mixed encrypted connection" and that the shield icon is gray
